src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styles) {
+  const styleObject = {};
+
+  // Поділ  рядків крапками з комою
+  const declarations = styles.split(';');
+
+  declarations.forEach((declaration) => {
+    // Розділ кожну декларацію двокрапкою
+    const [property, ...values] = declaration.split(':');
```

Comment:
The use of the rest operator (`...values`) to capture all values after the first colon is a clever way to handle potential edge cases with multiple colons in the property value. This ensures that the entire value is captured correctly.